infer_structcol/inference.py

```diff
@@ -22,25 +23,41 @@ def find_max_like(data, sample, seed=None):
         experimental dataset
     sample: Sample object
         information about the sample that produced data_spectrum
-    phi_guess: float
-        user's best guess of the expected volume fraction
     seed: int (optional)
-        if specified, passes the seed through to the MC multiple scatterin 
+        if specified, passes the seed through to the MC multiple scattering 
         calculation
-
+    theta_guess: list of floats (optional)
+        user's best guess of the expected parameter values (phi, radius, 
+        thickness)
+    theta_range: 2 by 3 array of floats 
```

Comment:
There are default values for theta_range that are at this point hard-coded, but these encompass the entire range of possible values (min_phi = 0.1, max_phi = 0.74, min_radius = 10, max_radius = 1000, min_thickness = 1, max_thickness = 1000). Most of the time, the user will have better information about the system, so they can set their own values for the theta_range. But if they don't input values for the theta_range, then the code will calculate the MAP with the default hard-coded values.